app/jobs/geocode_location_job.rb

```diff
@@ -1,4 +1,6 @@
 class GeocodeLocationJob < ApplicationJob
+  class NoResultsFound < StandardError; end
```

Comment:
It seems funny to me to have an `Error` class that never gets thrown, but I do see how this gets the job done below…